<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveAppearance</class>
 <widget class="QWidget" name="CurveAppearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>171</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox9_2_2_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="5">
       <widget class="QLabel" name="_textLabelWeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Weight:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_spinBoxLineWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QComboBox" name="_comboPointSymbol">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Point Type</string>
        </property>
        <property name="whatsThis">
         <string>If &quot;Show Points&quot; is checked, the selected point type will be drawn at each data point.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QComboBox" name="_comboLineStyle">
        <property name="toolTip">
         <string>Line Type</string>
        </property>
        <property name="whatsThis">
         <string>If &quot;Show Lines&quot; is checked, the selected line style.</string>
        </property>
        <property name="si" stdset="0">
         <string notr="true">line style</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="3">
       <widget class="QComboBox" name="_comboHeadSymbol"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>16</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="_showLines">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Check this to cause lines to be drawn between data points.</string>
        </property>
        <property name="text">
         <string>&amp;Lines</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="_showPoints">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Check this to cause the selected points to be drawn at each data point.</string>
        </property>
        <property name="text">
         <string>Po&amp;ints</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QCheckBox" name="_showHead">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Head</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="_showBars">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Check this to cause lines to be drawn between data points.</string>
        </property>
        <property name="text">
         <string>&amp;Bargraph</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Kst::ColorButton" name="_color">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="si" stdset="0">
         <string notr="true">curve color</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="Kst::ColorButton" name="_barFillColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="si" stdset="0">
         <string notr="true">curve bar fill</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QSpinBox" name="_spinBoxLineWidth">
        <property name="toolTip">
         <string>Line Width</string>
        </property>
        <property name="whatsThis">
         <string>If &quot;Show Lines&quot; is checked, the selected line width.</string>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLabel" name="_textLabelPointSize">
        <property name="text">
         <string>&amp;Size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>_spinBoxPointSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QLabel" name="_textLabelPointDensity">
        <property name="text">
         <string>Point &amp;Density:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_comboPointDensity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="_comboPointDensity">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Point Type</string>
        </property>
        <property name="whatsThis">
         <string>If &quot;Show Points&quot; is checked, the selected point type will be drawn at each data point.</string>
        </property>
        <item>
         <property name="text">
          <string comment="show every point">All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="high density of points">High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="medium density of points">Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="low density of points">Low</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="Kst::ColorButton" name="_headColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="si" stdset="0">
         <string notr="true">curve head color</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QDoubleSpinBox" name="_spinBoxPointSize"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_showLines</tabstop>
  <tabstop>_comboLineStyle</tabstop>
  <tabstop>_spinBoxLineWidth</tabstop>
  <tabstop>_color</tabstop>
  <tabstop>_showPoints</tabstop>
  <tabstop>_comboPointSymbol</tabstop>
  <tabstop>_comboPointDensity</tabstop>
  <tabstop>_showHead</tabstop>
  <tabstop>_comboHeadSymbol</tabstop>
  <tabstop>_headColor</tabstop>
  <tabstop>_showBars</tabstop>
  <tabstop>_barFillColor</tabstop>
 </tabstops>
 <includes>
  <include location="global">kst_export.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
